curriculum/challenges/english/25-front-end-development/workshop-cafe-menu/5f3c866dbf362f99b9a0c6d0.md

```diff
@@ -21,21 +21,21 @@ You should use the `.item p` selector.
 
 ```js
 const hasItemP = new __helpers.CSSHelp(document).getStyle('.item p');
```

Comment:
```suggestion
const itemPStyle = new __helpers.CSSHelp(document).getStyle('.item p');
```

let's also change the variable name so it makes more sense. variable name should also be changed on line 24